cms/djangoapps/contentstore/signals/handlers.py

```diff
@@ -20,6 +20,7 @@
 from xmodule.modulestore.django import SignalHandler, modulestore
 
 from .signals import GRADING_POLICY_CHANGED
+from . import course_dates  # Register the course_date handlers
```

Comment:
I see that you import it here but idk for me it would better all in one place.  it would be easy for me to overlook this import.  but just my opinion

Reply:
Yeah, I wish there were a better way to have multiple handlers, but I think it's better to keep it here than to put it in the same place as the code that currently imports `handlers.py`.